src/main/kotlin/org/pecasonline/NovoPecasOnlineApplication.kt

```
@@ -22,7 +22,7 @@ class NovoPecasOnlineApplication {
     fun sentryTest(): CommandLineRunner {
         return CommandLineRunner {
             try {
-                throw Exception("This is a test.")
+                throw Exception("Oi Meu chapa")
```

detekt (1.23.8)

[warning] Exception is a too generic Exception. Prefer throwing specific exceptions that indicate a specific error case.

(detekt.exceptions.TooGenericExceptionThrown)